app/services/reductions-service.js

```diff
@@ -15,11 +15,11 @@ module.exports.getReductions = function (id, organisationLevel) {
   var result = {}
   var organisationalUnitType = getOrganisationUnit('name', organisationLevel)
 
-  var getReductionsDataPromise = getReductions(id)
-
   result.breadcrumbs = getBreadcrumbs(id, organisationLevel)
   result.title = result.breadcrumbs[0].title
   result.subTitle = organisationalUnitType.displayText
+
+  var getReductionsDataPromise = getReductions(id)
```

Comment:
What was the problem here, why did this need moved?  You could probably remove this Promise variable and start the return below on getReductions(id).  Bit cleaner In my opinion

Reply:
Happy to move. Not completely sure why the ordering needed changed, but it fixed the failing test